### DISCUSSION TITLE
[Feature] schedule machine learning instead of running after upload

### DISCUSSION THREAD

**Discussion by JustC0des** [immich-app/immich#14961] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I am running my Immich on an Raspberry Pi and when uploading a whole lot of images, the host can be quite occupied. Sometimes this is then also shown in the app’s performance of Immich (images not loading etc.) if other processes are running on the host. 
It might make sense to have a possibility to not only disable machine learning completely before uploading a lot of assets, instead a option to choose a cron-based schedule similar to the postgres backup method makes sense. The Jobs therefore do not need to be run manually by the admin. Machine learning then has new states. On, off, and scheduled. The scheduled method then performs the machine learning on the missing assets whenever the cron schedule is met (for example each day on midnight). 
This gives more flexibility and reliability on the system as it transfers the system load to times where the usage is anyways not high or even null, which is especially on low scaled environments of a high importance!

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#5498
